db/compaction/tiered_compaction_test.cc

```diff
@@ -1249,8 +1249,11 @@ TEST_F(PrecludeLastLevelTest, MigrationFromPreserveTimeManualCompaction) {
   options.num_levels = kNumLevels;
   DestroyAndReopen(options);
 
+  // bootstap DB sequence numbers (FIXME: make these steps unnecessary)
+  ASSERT_OK(Put("foo", "bar"));
```

Comment:
nit: bootstrap

Curious why do we need to add this bootstrap step?

Reply:
It makes the DB last sequence number != 0 so that it will record seqno->time mapping. It refuses seqno==0 for sanity checking purposes.